other/get.webgl.org/troubleshooting/DoNotCopyOrLinkThisFileElseYouWillNotGetAutoUpdatedHelpForYourUsers.js

```diff
@@ -148,35 +148,35 @@ var BrowserDetect = {
     subString: "Win",
     identity: "Windows",
     browsers: [
-      {url: "http://www.mozilla.com/en-US/firefox/new/", name: "Mozilla Firefox"},
-      {url: "http://www.opera.com/", name: "Opera"},
-      {url: "http://www.google.com/chrome/", name: "Google Chrome"},
-      {url: "http://www.microsoft.com/en-us/windows/windows-10-upgrade ", name: "Edge"},
+      {url: "https://www.mozilla.org/en-US/firefox/new/", name: "Mozilla Firefox"},
+      {url: "https://www.opera.com", name: "Opera"},
+      {url: "https://www.google.com/chrome/", name: "Google Chrome"},
+      {url: "https://www.microsoft.com/en-us/windows/get-windows-10", name: "Edge"},
       {url: "http://www.microsoft.com/ie", name: "Internet Explorer"}
     ]
   },
   { string: navigator.platform,
     subString: "Mac",
     identity: "Mac",
     browsers: [
-      {url: "http://www.mozilla.com/en-US/firefox/new/", name: "Mozilla Firefox"},
-      {url: "http://www.google.com/chrome/", name: "Google Chrome"},
-      {url: "http://www.opera.com/", name: "Opera"},
-      {url: "http://www.webkit.org/", name: "WebKit Developer Builds"}
+      {url: "https://www.mozilla.org/en-US/firefox/new/", name: "Mozilla Firefox"},
+      {url: "https://www.google.com/chrome/", name: "Google Chrome"},
+      {url: "https://www.opera.com", name: "Opera"},
+      {url: "https://webkit.org", name: "WebKit Developer Builds"}
     ]
   },
   { string: navigator.userAgent,
     subString: "iPhone",
     identity: "iPhone/iPod",
     browsers: [
-      {url: "http://www.mozilla.com/en-US/firefox/new/", name: "Mozilla Firefox"}
+      {url: "https://www.mozilla.org/en-US/firefox/new/", name: "Mozilla Firefox"}
```

Comment:
Chrome's now available on iOS too, but I don't know what a good URL would be to link to.

Reply:
I'd guess chrome.com would probably be fine, alternatively the qr code on <https://www.google.com/chrome/thank-you.html> goes to <http://google.com/chrome/go-mobile>